### PR TITLE
Configure long polling activate jobs handler

### LINKED ISSUE #8777: As a user I want to configure `LongPollingActivateJobsHandler`
**Is your feature request related to a problem? Please describe.**
It would be nice to have the ability to customize `LongPollingActivateJobsHandler`. Like timeouts, minimal empty responses, etc.

**Describe the solution you'd like**
I think that these settings should be added via Spring Boot configuration **and** environment variables.

**Describe alternatives you've considered**
Maybe these should be configured via environment variables **or** Spring Boot configuration.

**Additional context**
You can find the source of this issue here: https://camunda-cloud.slack.com/archives/C6WGNHV2A/p1644477175720079

### PR DESCRIPTION
**Pull request by EuroLew**
## Description
Added Long polling configuration options and associated documentation.

## Related issues

closes #8777

### CODE REVIEW

File: dist/src/main/config/gateway.yaml.template [hunk 1]
```diff
@@ -231,6 +231,18 @@
       # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_ENABLED.
       # enabled: true
 
+      # Set the timeout for long polling
```
Comment (oleschoenburg) [inline dist/src/main/config/gateway.yaml.template:234]: :wrench: Specify the unit here: milliseconds.

File: dist/src/main/config/gateway.yaml.template [hunk 2]
```diff
@@ -231,6 +231,18 @@
...
+      # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_TIMEOUT.
+      # timeout: 10000
+
+      # Set the probe timeout for long polling
```
Comment (oleschoenburg) [inline dist/src/main/config/gateway.yaml.template:238]: :wrench: Specify the unit here: milliseconds.

File: dist/src/main/config/gateway.yaml.template [hunk 3]
```diff
@@ -231,6 +231,18 @@
...
+      # This setting can also be overridden using the environment variable ZEEBE_GATEWAY_LONGPOLLING_PROBETIMEOUT.
+      # probeTimeout: 10000
+
+      # Set the number of minimum empty responses
```
Comment (oleschoenburg) [inline dist/src/main/config/gateway.yaml.template:242]: :wrench: Explain a bit more what that means. For example, just copy a  code comment from `LongPollingActivateJobsHandler`:
> Minimum number of responses with jobCount 0 to infer that no jobs are available

**Review by lenaschoenburg — APPROVED** [review]
Looks good but I suggested a couple of improvements.

:wrench: Please add documentation (in a separate PR) here: https://github.com/camunda/camunda-docs/blob/main/docs/self-managed/zeebe-deployment/configuration/gateway.md

**Pull request merged**